docs/Formula-Cookbook.md

```diff
@@ -251,7 +251,36 @@ uses_from_macos "curl", since: :monterey
 
 Homebrew doesn’t package already-packaged language-specific libraries. These should be installed directly from `gem`/`cpan`/`pip` etc.
 
-If you're installing an application then use [`resource`](https://rubydoc.brew.sh/Formula#resource-class_method)s for all language-specific dependencies:
+### Ruby Gem Dependencies
+
+The preferred mechanism for installing gem dependencies is to use `bundler` with the upstream's `Gemfile.lock`. This requires the upstream checks in their `Gemfile.lock`, so if they don't, it's a good idea to file a bug and ask them to do so. Assuming they have one, this is as simple as:
+
+```ruby
+ENV["GEM_HOME"] = libexec
+system "bundle", "install"
```

Comment:
```suggestion
system "bundle", "install", "--without", "development"
```

Reply:
On side note, perhaps these should be included as commented out lines in `brew create --ruby`. I feel that is often the easiest way to showcase our preferences (though I don't know statistics of how many people actually use that command).

Reply:
I didn't even know there WAS a --python and a --ruby. I'll add that to the docs in this PR.